content/francais/python-turtle/the-basics.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Les bases - Dessiner une ligne colorée"
+date: 2024-09-17T13:24:17-07:00
+weight: 2
+draft: false
+---
+
+Avant qu'Alex ne puisse apprendre à créer une ruche, il doit d'abord apprendre à dessiner une ligne colorée. Nous ajouterons du code à la fenêtre Trinket ci-dessous, et nous le ferons en deux parties.
+
+Tout d'abord, nous devons dire à la tortue quelle couleur nous voulons que la ligne soit. Pour cela, tapons :
+
+``` python
+turtle.color("orange")
+```
+
+Essayons de comprendre cette ligne de code plus en détail.
+
+{{% notice tip %}}
+
+## Chaînes de caractères
+
+Tout ce que vous voyez entre **guillemets** s'appelle une **chaîne de caractères**. Une chaîne est simplement une séquence de caractères (qu'il s'agisse de lettres, de chiffres ou de symboles).
+
+Dans le code ci-dessus, `"orange"` est une chaîne de caractères. D'autres exemples de chaînes incluent `"123"`, `"abc!"` et `"green"`. Attention, `123`, `abc!` et `green` ne sont pas des chaînes car il manque des guillemets !
+
+## Méthodes
+
+`turtle.color()` est une **méthode.** Nous utilisons des méthodes pour que la tortue effectue certaines actions. Dans ce cas, `turtle.color()` change la couleur des lignes qu'elle dessine selon la couleur que nous lui indiquons. Nous lui avons dit de changer la couleur en orange en lui envoyant la **chaîne** `"orange"` comme **argument**. 
+
+## Arguments
+
+Les arguments sont des entrées que vous passez aux **méthodes** pour utiliser. Dans notre exemple, `turtle.color()` a besoin d'un argument de type **chaîne** représentant le nom de la couleur à définir pour la tortue, donc nous avons passé `"orange"` comme **argument** à la méthode. Nous aurions aussi pu lui dire de changer la couleur en quelque chose d'autre, mais il fallait que ce soit une couleur que `turtle.color()` comprend. Les couleurs qu'il reconnaît sont : `"red"`, `"orange"`, `"yellow"`, `"green"`, `"blue"`, `"purple"`, `"black"`, `"pink"`, et `"gray"`. Si nous envoyons autre chose qu'une de ces chaînes de couleurs, nous obtenons la couleur par défaut, qui est le noir.
```

Comment:
In "Dans notre exemple, `turtle.color()` a besoin d'un argument de type **chaîne** " change to

"Dans notre exemple, `turtle.color()` a besoin d'un argument de type **string**"